ext/js/pages/settings/dictionary-controller.js

```diff
@@ -849,7 +851,7 @@ export class DictionaryController {
         if (typeof title !== 'string') { return; }
         delete modal.node.dataset.dictionaryTitle;
 
-        void this._deleteDictionary(title);
+        void this._enqueueDictionaryDelete(title);
```

Comment:
Is it possible to call this multiple times, causing a race condition between multiple invocations?

Reply:
No, if the dictionary is in queue, the delete button will be disabled.